### ISSUE TITLE
Module Unable to See Timer/Clock Variables When Using ProPresenter 7.9.1 beta 

### ISSUE THREAD

**Issue by jhalb0129** [bitfocus/companion-module-renewedvision-propresenter#125]
I use Companion 2.2.0 and triggers to execute actions based on clock/timer variables in ProPresenter. For several months, the clock variables in companion have been very solid. However, when I upgraded to the Pro7 7.9.1 beta today, I noticed that none of the clocks populated in companion. The variables are simply missing from the list in the companion GUI. I tried downgrading 7.9 (official release) and they all came back. Starting with 7.9, Renewed Vision has started publishing an API, and I wonder if the 7.9.1 beta makes changes to the clock API, or if it is a bug in the program, which should be reported to Renewed Vision. Could someone confirm one way or another? Any insight would be appreciated. Thanks!

**Comment by greyshirtguy** [comment]
Sorry - I've only just seen this today - been a bit tied up...
I'll look into it!

**Comment by greyshirtguy** [comment]
yeah - it seems to be a bug in pro 7.9.1 on MacOS (still works on 7.9.1/Wiondows)
I have raised it with product owner....
It's unsupported - but it also affects the offical (supported) remote app - so hopefully they agree to spend some time and fix it soon!

**Comment by paulstec** [comment]
It looks like RV has published the new API as noted above.  The module has an experimental section that looks like it is for the new API.

Are there plans to have this module support the new API?  If so, is there an anticipated time for an update?
